### ISSUE TITLE
Static file service

### ISSUE THREAD

**Issue by Grahack** [tinspin/rupy#1]
Hi,
Is it possible to use rupy like a static file server, the root being where the run.bat file was launched?
Thanks.

**Comment by tinspin** [comment]
Yes, but the root is app/content from run.bat... You can always hack it!

**Comment by Grahack** [comment]
OK. If I manage to add a command line option, like `--root ./path/to/root`, I'll make a pull request.

Petit coup de pub au passage: http://microalg.info/
